Oct-22 02:16:12.675 [main] DEBUG nextflow.cli.Launcher - $> nextflow run main.nf -dump-channels -profile singularity --input ./example/samplelist.csv --genome GRCh38 --tools dcc
Oct-22 02:16:12.769 [main] INFO  nextflow.cli.CmdRun - N E X T F L O W  ~  version 21.10.6
Oct-22 02:16:12.794 [main] INFO  nextflow.cli.CmdRun - Launching `main.nf` [determined_poisson] - revision: 2db6276e14
Oct-22 02:16:12.807 [main] DEBUG nextflow.config.ConfigBuilder - Found config local: /restricted/projectnb/casa/mtLin/DCC_nextflow/nextflow.config
Oct-22 02:16:12.808 [main] DEBUG nextflow.config.ConfigBuilder - Parsing config file: /restricted/projectnb/casa/mtLin/DCC_nextflow/nextflow.config
Oct-22 02:16:12.831 [main] DEBUG nextflow.config.ConfigBuilder - Applying config profile: `singularity`
Oct-22 02:16:14.218 [main] DEBUG nextflow.config.ConfigBuilder - Available config profiles: [debug, conda, singularity, docker]
Oct-22 02:16:14.261 [main] DEBUG nextflow.plugin.PluginsFacade - Setting up plugin manager > mode=prod; plugins-dir=/usr3/graduate/minty/.nextflow/plugins
Oct-22 02:16:14.262 [main] DEBUG nextflow.plugin.PluginsFacade - Plugins default=[]
Oct-22 02:16:14.273 [main] INFO  org.pf4j.DefaultPluginStatusProvider - Enabled plugins: []
Oct-22 02:16:14.274 [main] INFO  org.pf4j.DefaultPluginStatusProvider - Disabled plugins: []
Oct-22 02:16:14.277 [main] INFO  org.pf4j.DefaultPluginManager - PF4J version 3.4.1 in 'deployment' mode
Oct-22 02:16:14.285 [main] INFO  org.pf4j.AbstractPluginManager - No plugins
Oct-22 02:16:14.333 [main] DEBUG nextflow.Session - Session uuid: 54073d0b-766b-4ba6-a1c4-ddb2a8a82f7f
Oct-22 02:16:14.334 [main] DEBUG nextflow.Session - Run name: determined_poisson
Oct-22 02:16:14.334 [main] DEBUG nextflow.Session - Executor pool size: 28
Oct-22 02:16:14.352 [main] DEBUG nextflow.cli.CmdRun - 
  Version: 21.10.6 build 5660
  Created: 21-12-2021 16:55 UTC (11:55 EDT)
  System: Linux 4.18.0-553.22.1.el8_10.x86_64
  Runtime: Groovy 3.0.9 on Java HotSpot(TM) 64-Bit Server VM 1.8.0_411-b09
  Encoding: UTF-8 (UTF-8)
  Process: 2353120@scc-q18.scc.bu.edu [192.168.18.202]
  CPUs: 28 - Mem: 187 GB (39.7 GB) - Swap: 8 GB (8 GB)
Oct-22 02:16:14.383 [main] DEBUG nextflow.Session - Work-dir: /restricted/projectnb/casa/mtLin/DCC_nextflow/work [nfs]
Oct-22 02:16:14.396 [main] DEBUG nextflow.executor.ExecutorFactory - Extension executors providers=[]
Oct-22 02:16:14.405 [main] DEBUG nextflow.Session - Observer factory: DefaultObserverFactory
Oct-22 02:16:14.446 [main] DEBUG nextflow.util.CustomThreadPool - Creating default thread pool > poolSize: 29; maxThreads: 1000
Oct-22 02:16:14.518 [main] DEBUG nextflow.Session - Session start invoked
Oct-22 02:16:14.822 [main] DEBUG nextflow.script.ScriptRunner - > Launching execution
Oct-22 02:16:15.583 [main] DEBUG nextflow.executor.ExecutorFactory - << taskConfig executor: null
Oct-22 02:16:15.583 [main] DEBUG nextflow.executor.ExecutorFactory - >> processorType: 'local'
Oct-22 02:16:15.587 [main] DEBUG nextflow.executor.Executor - [warm up] executor > local
Oct-22 02:16:15.593 [main] DEBUG n.processor.LocalPollingMonitor - Creating local task monitor for executor 'local' > cpus=28; memory=187 GB; capacity=28; pollInterval=100ms; dumpInterval=5m
Oct-22 02:16:15.617 [main] DEBUG nextflow.util.CustomThreadPool - Creating default thread pool > poolSize: 29; maxThreads: 1000
Oct-22 02:16:15.806 [main] DEBUG nextflow.plugin.PluginUpdater - Installing plugin nf-amazon version: 1.3.4
Oct-22 02:16:15.832 [main] INFO  org.pf4j.AbstractPluginManager - Plugin 'nf-amazon@1.3.4' resolved
Oct-22 02:16:15.833 [main] INFO  org.pf4j.AbstractPluginManager - Start plugin 'nf-amazon@1.3.4'
Oct-22 02:16:16.293 [main] DEBUG nextflow.plugin.BasePlugin - Plugin started nf-amazon@1.3.4
Oct-22 02:16:16.313 [main] DEBUG nextflow.file.FileHelper - > Added 'S3FileSystemProvider' to list of installed providers [s3]
Oct-22 02:16:16.313 [main] DEBUG nextflow.file.FileHelper - Started plugin 'nf-amazon' required to handle file: s3://ngi-igenomes/igenomes/Homo_sapiens/NCBI/GRCh38/Sequence/WholeGenomeFasta/genome.fa
Oct-22 02:16:16.349 [main] DEBUG nextflow.file.FileHelper - Creating a file system instance for provider: S3FileSystemProvider
Oct-22 02:16:16.354 [main] DEBUG nextflow.file.FileHelper - AWS S3 config details: {max_error_retry=5}
Oct-22 02:16:16.783 [main] DEBUG nextflow.Session - Workflow process names [dsl2]: DCC_MATE2_1ST_PASS, SAMPLESHEET_CHECK, DCC_MATE1_2ND_PASS, DCC, DCC_1ST_PASS, DCC_MATE2_2ND_PASS, DCC_MATE1_1ST_PASS, DCC_MATE2_SJDB, DCC_2ND_PASS, SJDB, INPUT_CHECK:SAMPLESHEET_CHECK, DCC_MATE1_SJDB, DCC_FILTER, DCC_SJDB, STAR_ALIGN
Oct-22 02:16:16.789 [main] WARN  nextflow.Session - There's no process matching config selector: STAR_GENOMEGENERATE
Oct-22 02:16:16.791 [main] DEBUG nextflow.extension.CH - Bridging dataflow queue=DataflowQueue(queue=[])
Oct-22 02:16:16.792 [main] DEBUG nextflow.Session - Ignite dataflow network (4)
Oct-22 02:16:16.793 [main] DEBUG nextflow.processor.TaskProcessor - Starting process > INPUT_CHECK:SAMPLESHEET_CHECK
Oct-22 02:16:16.793 [main] DEBUG nextflow.script.ScriptRunner - > Await termination 
Oct-22 02:16:16.793 [main] DEBUG nextflow.Session - Session await
Oct-22 02:16:16.873 [Actor Thread 17] DEBUG nextflow.container.SingularityCache - Singularity found local store for image=https://depot.galaxyproject.org/singularity/python:3.8.3; path=/restricted/projectnb/casa/mtLin/nextflow_image/depot.galaxyproject.org-singularity-python-3.8.3.img
Oct-22 02:16:16.976 [Task submitter] DEBUG nextflow.executor.LocalTaskHandler - Launch cmd line: /bin/bash -ue .command.run
Oct-22 02:16:16.980 [Task submitter] INFO  nextflow.Session - [cf/3b5f57] Submitted process > INPUT_CHECK:SAMPLESHEET_CHECK (samplelist.csv)
Oct-22 02:16:17.266 [Task monitor] DEBUG n.processor.TaskPollingMonitor - Task completed > TaskHandler[id: 1; name: INPUT_CHECK:SAMPLESHEET_CHECK (samplelist.csv); status: COMPLETED; exit: 127; error: -; workDir: /restricted/projectnb/casa/mtLin/DCC_nextflow/work/cf/3b5f57efdefd163bc9d451088ceffd]
Oct-22 02:16:17.283 [Task monitor] ERROR nextflow.processor.TaskProcessor - Error executing process > 'INPUT_CHECK:SAMPLESHEET_CHECK (samplelist.csv)'

Caused by:
  Process `INPUT_CHECK:SAMPLESHEET_CHECK (samplelist.csv)` terminated with an error exit status (127)

Command executed:

  check_samplesheet.py \
      samplelist.csv \
      samplesheet.valid.csv
  
  cat <<-END_VERSIONS > versions.yml
  "INPUT_CHECK:SAMPLESHEET_CHECK":
      python: $(python --version | sed 's/Python //g')
  END_VERSIONS

Command exit status:
  127

Command output:
  (empty)

Command error:
  .command.sh: line 3: check_samplesheet.py: command not found

Work dir:
  /restricted/projectnb/casa/mtLin/DCC_nextflow/work/cf/3b5f57efdefd163bc9d451088ceffd

Tip: when you have fixed the problem you can continue the execution adding the option `-resume` to the run command line
Oct-22 02:16:17.288 [main] DEBUG nextflow.Session - Session await > all process finished
Oct-22 02:16:17.289 [Task monitor] DEBUG nextflow.Session - Session aborted -- Cause: Process `INPUT_CHECK:SAMPLESHEET_CHECK (samplelist.csv)` terminated with an error exit status (127)
Oct-22 02:16:17.299 [main] DEBUG nextflow.Session - Session await > all barriers passed
Oct-22 02:16:17.304 [main] DEBUG nextflow.trace.WorkflowStatsObserver - Workflow completed > WorkflowStats[succeededCount=0; failedCount=1; ignoredCount=0; cachedCount=0; pendingCount=0; submittedCount=0; runningCount=0; retriesCount=0; abortedCount=0; succeedDuration=0ms; failedDuration=246ms; cachedDuration=0ms;loadCpus=0; loadMemory=0; peakRunning=1; peakCpus=1; peakMemory=0; ]
Oct-22 02:16:17.497 [main] DEBUG nextflow.CacheDB - Closing CacheDB done
Oct-22 02:16:17.497 [main] INFO  org.pf4j.AbstractPluginManager - Stop plugin 'nf-amazon@1.3.4'
Oct-22 02:16:17.497 [main] DEBUG nextflow.plugin.BasePlugin - Plugin stopped nf-amazon
Oct-22 02:16:17.505 [main] DEBUG nextflow.script.ScriptRunner - > Execution complete -- Goodbye
